File: Domain/EnlistedAideCheckList.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain
{
    public class EnlistedAideCheckList
    {
        public Guid Id { get; set; }
        public Guid ActivityId { get; set; }
        public Activity Activity { get; set; }
        public bool AlcoholEstimate { get; set; }
        public bool PrepareLegalReview {get; set;}
        public bool SendToLegalForApproval { get; set; }
        public bool PreparePRAForm { get; set; }    
        public bool PrepareGuestList { get; set; }
        public bool Prepare4843GuestList { get; set; }
        public bool PrepareMenu { get; set; }
        public bool MenuReviewedByPrincipal { get; set; }
        public bool OrderAlcohol { get; set; }
        public bool GFEBS { get; set; }
        public bool GatherIce { get; set; }
        public bool SweepAndMop { get; set; }
        public bool HighTopsAndTables { get; set; }
        public bool PolishSilver { get; set; }
        public bool CleanCutlery { get; set; }
        public bool CleanPlates { get; set; }
        public bool CleanServiceItems { get; set; }
        public bool NapkinsPressed { get; set; }
        public bool NapkinsRolled { get; set; }
        public bool FoodPrep { get; set; }
        public bool Dust { get; set; }
        public bool Cook { get; set; }
        public bool Coffee { get; set; }
        public bool IceBeverages { get; set; }
        public bool Sterno { get; set; }
        public bool FoodShopping {get; set;}
        public bool TentSetUp {get; set;}

    }
}

File: client-app/src/app/models/enlistedAideChecklist.ts

export interface EnlistedAideChecklist{
    id: string
    activityId: string
    alcoholEstimate: boolean
    sendToLegalForApproval: boolean
    prepareLegalReview: boolean
    preparePRAForm: boolean
    prepareGuestList: boolean
    prepare4843GuestList: boolean
    prepareMenu: boolean
    menuReviewedByPrincipal: boolean
    orderAlcohol: boolean
    gfebs: boolean
    gatherIce: boolean
    sweepAndMop: boolean
    highTopsAndTables: boolean
    polishSilver: boolean
    cleanCutlery: boolean
    cleanPlates: boolean
    cleanServiceItems: boolean
    napkinsPressed: boolean
    napkinsRolled: boolean
    foodPrep: boolean
    dust: boolean
    cook: boolean
    coffee: boolean
    iceBeverages: boolean
    sterno: boolean
    foodShopping: boolean
    tentSetUp: boolean

}

export class EnlistedAideChecklist implements EnlistedAideChecklist{
    constructor(init?: EnlistedAideChecklistFormValues){
      Object.assign(this, init);
    }
}

export class EnlistedAideChecklistFormValues{
    id: string = '';
    activityId: string = '';
    alcoholEstimate: boolean = false;
    sendToLegalForApproval: boolean = false;
    prepareLegalReview: boolean = false;
    preparePRAForm: boolean = false;
    prepareGuestList: boolean = false;
    prepare4843GuestList: boolean = false;
    prepareMenu: boolean = false;
    menuReviewedByPrincipal: boolean = false;
    orderAlcohol: boolean = false;
    gfebs: boolean = false;
    gatherIce: boolean = false;
    sweepAndMop: boolean = false;
    highTopsAndTables: boolean = false;
    polishSilver: boolean = false;
    cleanCutlery: boolean = false;
    cleanPlates: boolean = false;
    cleanServiceItems: boolean = false;
    napkinsPressed: boolean = false;
    napkinsRolled: boolean = false;
    foodPrep: boolean = false;
    dust: boolean = false;
    cook: boolean = false;
    coffee: boolean = false;
    iceBeverages: boolean = false;
    sterno: boolean = false;
    foodShopping: boolean = false;
    tentSetUp: boolean = false;

    constructor(enlistedAideChecklist?: EnlistedAideChecklistFormValues){
        if(enlistedAideChecklist){
            this.id = enlistedAideChecklist.id;
            this.activityId = enlistedAideChecklist.activityId;
            this.alcoholEstimate = enlistedAideChecklist.alcoholEstimate;
            this.sendToLegalForApproval = enlistedAideChecklist.sendToLegalForApproval;
            this.prepareLegalReview = enlistedAideChecklist.prepareLegalReview;
            this.preparePRAForm = enlistedAideChecklist.preparePRAForm;
            this.prepareGuestList = enlistedAideChecklist.prepareGuestList;
            this.prepare4843GuestList = enlistedAideChecklist.prepare4843GuestList;
            this.prepareMenu = enlistedAideChecklist.prepareMenu;
            this.menuReviewedByPrincipal = enlistedAideChecklist.menuReviewedByPrincipal;
            this.orderAlcohol = enlistedAideChecklist.orderAlcohol;
            this.gfebs = enlistedAideChecklist.gfebs;
            this.gatherIce = enlistedAideChecklist.gatherIce;
            this.sweepAndMop = enlistedAideChecklist.sweepAndMop;
            this.highTopsAndTables = enlistedAideChecklist.highTopsAndTables;
            this.polishSilver = enlistedAideChecklist.polishSilver;
            this.cleanCutlery = enlistedAideChecklist.cleanCutlery;
            this.cleanPlates = enlistedAideChecklist.cleanPlates;
            this.cleanServiceItems = enlistedAideChecklist.cleanServiceItems;
            this.napkinsPressed = enlistedAideChecklist.napkinsPressed;
            this.napkinsRolled = enlistedAideChecklist.napkinsRolled;
            this.foodPrep = enlistedAideChecklist.foodPrep;
            this.dust = enlistedAideChecklist.dust;
            this.cook = enlistedAideChecklist.cook;
            this.coffee = enlistedAideChecklist.coffee;
            this.iceBeverages = enlistedAideChecklist.iceBeverages;
            this.sterno = enlistedAideChecklist.sterno;
            this.foodShopping = enlistedAideChecklist.foodShopping;
            this.tentSetUp = enlistedAideChecklist.tentSetUp;
        }
    }
}